tests/informatics/test_predictors.py

```diff
@@ -343,7 +343,7 @@ def test_auto_ml_multiple_outputs(auto_ml_multiple_outputs):
 def test_ing_to_formulation_initialization(ing_to_formulation_predictor):
     """Make sure the correct fields go to the correct places for an ingredients to formulation predictor."""
     assert ing_to_formulation_predictor.name == 'Ingredients to formulation predictor'
-    assert ing_to_formulation_predictor.output.key == 'formulation'
+    assert ing_to_formulation_predictor.output.key == 'Formulation'
```

Comment:
Can this be a constant / enum instead of having that string appear several times in tests?

Reply:
Sure, I can make an enum of the two allowed keys.